src/integration/aggregator/aggregator.go

```diff
@@ -128,9 +175,9 @@ func testAggregatedGraphiteMetric(t *testing.T, m3 resources.M3Resources) {
 		}
 	}()
 
-	require.NoError(t, resources.Retry(func() error {
+	require.NoError(t, resources.RetryWithMaxTime(func() error {
 		return verifyGraphiteQuery(m3, carbonTarget, expectedCarbonMean)
-	}))
+	}, 2*time.Minute))
```

Comment:
Aggregation takes a slight bit longer now that we're using separate aggregators.